tests/test_utils.py

```diff
@@ -0,0 +1,15 @@
+import sys
+import os
+sys.path.append(os.path.dirpath(os.path.realpath(__file__)) + '/../')
```

Comment:
Is this an acceptable way to import the utils.py module from the outer directory? Seems like a hack to me, but this is what I got after some stackoverflow searches.

Reply:
I hadn't seen this comment 😄 
I've replied at https://github.com/marco-c/autowebcompat/pull/36#discussion_r166479187.

Reply:
@SkullTech @marco-c  I think its always better to avoid system calls. :) Once the structure of the directory is set, it will be easier. OOP structure favours imports with ease.

Can I structure the whole network.py into a nice class with appropriate functions ? What do you think ?

Reply:
> @SkullTech @marco-c I think its always better to avoid system calls. :) Once the structure of the directory is set, it will be easier. OOP structure favours imports with ease.

For now, we are leaning towards moving the `network.py` and `utils.py` modules into a `autowebcompat` directory and leaving the scripts (which are never used as modules) in the top-level directory. Discussion at https://github.com/marco-c/autowebcompat/pull/36#pullrequestreview-94547867.

> Can I structure the whole network.py into a nice class with appropriate functions ? What do you think ?

Sounds good to me, this should happen regardless of the directory structure.